### ISSUE TITLE
first line of application.yml is missing in the "Single Sign On With GitHub" chapter

### ISSUE THREAD

**Issue by csfulop** [spring-guides/tut-spring-boot-oauth2#122]
I have tried to follow the tutorial on https://spring.io/guides/tutorials/spring-boot-oauth2/
Created the app according to the "Single Sign On With GitHub" chapter but it was not working.
Then tried and compared with the code from the github repo and the first line of application.yml is missing from the tutorial. (it is OK in the src)

**Comment by dsyer** [comment]
Looks OK to me. Can you be more precise about where the mismatch is? I see this in `application.yml` (https://github.com/spring-guides/tut-spring-boot-oauth2/blob/master/simple/src/main/resources/application.yml):

```
spring:
  security:
    oauth2:
      client:
        registration:
          github:
            client-id: your-github-client-id
            client-secret: your-github-client-secret
```

and I see the identical snippet in https://github.com/spring-guides/tut-spring-boot-oauth2/blob/master/simple/README.adoc

**Comment by csfulop** [comment]
Hi, I have created a pull request with the fix

**Comment by dsyer** [comment]
Got it. Thanks.

**Comment by csfulop** [comment]
Sorry, my editor messed up the indentation with tabs and I haven't noticed because I was focusing only on the missing first line :(
This fix is in pull request #124
